icu4c/source/test/cintltst/custrtst.c

```diff
@@ -962,6 +962,16 @@ TestSurrogateSearching() {
     ) {
         log_err("error: one of the u_str[str etc](\"aba\") incorrectly finds something\n");
     }
+    /* Regression test for ICU-20684 Use-of-uninitialized-value in isMatchAtCPBoundary
+     * Condiction: search the same string while the first char is not an
```

Comment:
typo: Condiction --> Condition

(you can use git commit --amend to keep a single commit)